src/openforms/js/components/form/map.js

```diff
@@ -62,6 +62,80 @@ const EDIT_FORM_TABS = {
         HIDDEN,
         CLEAR_ON_HIDE,
         {...IS_SENSITIVE_DATA, defaultValue: true},
+        {
+          type: 'number',
+          key: 'defaultZoom',
+          label: 'Default zoom level.',
+          description: 'Default zoom level for the leaflet map.',
+          validate: {
+            min: 1,
+            max: 18,
+          },
+        },
+        {
+          type: 'columns',
+          input: false,
+          label: 'starting location',
+          key: 'startPop',
+          tooltip: 'The Latitude and Longitude of the starting point of the map component.',
+          defaultValue: [{lat: '', lng: ''}],
+          columns: [
+            {
+              components: [
+                {
+                  label: 'Latitude',
+                  key: 'lat',
+                  input: true,
+                  type: 'number',
+                  decimalLimit: 7,
+                  delimiter: false,
+                  requireDecimal: true,
+                  tableView: false,
+                  inputFormat: 'plain',
+                  truncateMultipleSpaces: false,
+                  defaultValue: 52.1326332,
```

Comment:
I don't think it's a good idea to provide this default value at the component level - it means that every newly added map component gets the default baked in, so if you want to change the global default, then you have to edit all the existing form definitions that have a map.

I would not provide a default value at all, and if the value is blank, that implies -> use the global default from the global configuration.